app/Resources/views/firebase/firebase-messaging-ws.js.twig

```diff
@@ -0,0 +1,18 @@
+importScripts('https://www.gstatic.com/firebasejs/3.9.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/3.9.0/firebase-messaging.js');
+
+firebase.initializeApp({
+    messagingSenderId: "{{ firebase_messaging_sender_id }}"
+});
+
+const messaging = firebase.messaging();
```

Comment:
You are including this file directly in your Twig templates aren't you? You should use Webpack instead, for minification and as you use ES6 features.